src/cudamatrix/cu-matrix-test.cc

```diff
@@ -1430,6 +1430,10 @@ static void UnitTestCuMatrixAddVecVec() {
 
 template<typename Real>
 static void UnitTestCuMatrixAddMatMatBatched() {
+  // Random stride is disabled as AddMatMatBatched requires consistent stride
+#if HAVE_CUDA == 1
+  CuDevice::Instantiate().SetRandomStrideMode(false);
```

Comment:
`old_mode` looks better although this function probably won't be called from anywhere else.